README.in.md

```diff
@@ -14,10 +14,11 @@ In the modern Minecraft era, with Fabric's community effort, modders have begun
 ## This list
 
 This list will list recommended OptiFine alternatives by me (LambdAurora), so it has some requirements for mods to be listed here:
- - They must be publicly available, if it's only accessible through Discord they are not eligible.
- - Mods must respect the Minecraft's EULA.
- - Mods must not refer to a cheat client.
+ - They must be publicly available, if they're only available on request they are not eligible.
+ - Mods must respect the Mojang EULA.
+ - Cheat ("PVP") clients (Badlion / Lunar / etc.) are not considered mods.
```

Comment:
Unsure.

The original line is more in line with what I meant, and I would consider this change as a new line.